spec/models/exports/formatters/custom_field_hierarchy_spec.rb

```diff
@@ -73,7 +73,7 @@
       end
 
       it "returns the label and short" do
```

Comment:
:red_circle: The test case name does not reflect expected behaviour anymore. No short is displayed anywhere.